clients/privacy-center/pages/consent.tsx

```diff
@@ -44,6 +44,7 @@ const Consent: NextPage = () => {
     const getUserConsents = async () => {
       if (!consentRequestId) {
         router.push("/");
+        return;
```

Comment:
Why are these returns needed? 🤔

Reply:
Previously the function would keep running even though the logic would all be unnecessary. I noticed the `fetch` on line 87 would fire even though the `verificationCode` was already known to be invalid.